src/nbl/asset/interchange/CGraphicsPipelineLoaderMTL.cpp

```diff
@@ -78,10 +78,8 @@ void CGraphicsPipelineLoaderMTL::initialize()
     }
 
     // default pipelines
-    system::ISystem::future_t<core::smart_refctd_ptr<system::CFileView>> future;
     constexpr std::string_view filename = "Nabla default MTL material";
-    m_assetMgr->getSystem()->createFileView(DUMMY_MTL_CONTENT, sizeof(DUMMY_MTL_CONTENT), system::IFile::ECF_READ, filename);
-    auto default_mtl_file = future.get();
+    auto default_mtl_file = m_assetMgr->getSystem()->createFileView(DUMMY_MTL_CONTENT, sizeof(DUMMY_MTL_CONTENT), system::IFile::ECF_READ, filename);
```

Comment:
this is ugly, make file view be able to create itself (why do you even need the system to create a file view!?_